doc/src/fix_charge_regulation.rst

```diff
@@ -0,0 +1,184 @@
+
+.. Yuan documentation master file, created by
+   sphinx-quickstart on Sat Jan 30 14:06:22 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+   tc387: Multiple text additions/changes, Feb 2 2021
+.. index:: fix fix_charge_regulation
```

Comment:
This is giving the fix name as fix_charge_regulation which is incorrect and thus produces an incorrect index.

Reply:
done.